### PR TITLE
HW1

### PR DESCRIPTION
**Pull request by DanSafdie**
I did it!

### CODE REVIEW

File: people/danny-safdie.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Danny Safdie
+-------------
+
+![Danny Safdie's photo](photos/danny.jpg)
```
Comment (dblock) [inline people/danny-safdie.md]: Rename the JPG to danny-safdie.jpg, there will be other Danny's.

**Comment by dblock** [comment]
You also need to add a link to README, please.

Make sure your commits are _squashed_ (1 commit), see http://code.dblock.org/2015/08/31/getting-out-of-your-first-git-mess.html if you need help.

**Comment by DanSafdie** [comment]
Thanks for the edits. How does it look now?

**Comment by dblock** [comment]
I don't see the changes. Did you forget to push them to Github?

**Comment by DanSafdie** [comment]
screen shot 2015-09-02 at 9 42 00 am

**Comment by DanSafdie** [comment]
I think I did but not entirely certain (new to github). the attached image
shows that I "added some commits" is that the same as pushing to git. I
also created a pull request after making and committing all the changes

[image: Inline image 1]

**Comment by dblock** [comment]
Now this looks good, but you should figure out how to _squash_ these commits in 1. My post above will definitely help you, try to work through it.

**Comment by DanSafdie** [comment]
Is that something I can do at this point or something that I should have
done before but is not too late? (apologies for my lack of background with
git - a lot of this terminology goes way over my head)

File: people/danny-safdie.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Danny Safdie
+-------------
+
+![Danny Safdie's photo](photos/danny-safdie.jpg)
```
Comment (dblock) [inline people/danny-safdie.md]: The picture file name doesn't match the actual file name, so the image is broken.

**Comment by dblock** [comment]
It's something you can always do, and should do now. Right now your change conflicts with what we have _upstream_ and cannot be merged.

**Comment by DanSafdie** [comment]
How does my pull request look now?

**Comment by dblock** [comment]
Looks great, merging.

**Pull request merged**